server/controllers/error.js

```diff
@@ -1,9 +1,9 @@
-const notFound = (req,res)=>{
-  res.status(404).send("<h1> 404!!  PAGE NOT FOUND</h1>")
-}
+const notFound = (req, res, err) => {
```

Comment:
err is the first parameter.
check this https://expressjs.com/en/guide/error-handling.html